docs/content/usage/functions.en.md

```diff
@@ -66,7 +66,18 @@ module.exports = async function(context) {
 }
 ```
 
-Let's say you want to update the function to output "Hello Fission" instead of "Hello world", you can update the source file and update the source code for function:
+Let's say you want to update the function to output "Hello Fission" instead of "Hello world". 
+
+```js
+$ fission fn get --name hello
```

Comment:
```
$ cat hello.js
```

Reply:
Can we just remove that line
```
$ fission fn get --name hello
```
 altogether and leave only the updated code ?

Reply:
Sure, it's ok.